pkg/webhook/virtualmachine/virtualmachine.go

```diff
@@ -77,17 +77,19 @@ func (a *virtualMachineAnnotator) Handle(ctx context.Context, req admission.Requ
 
 	logger.V(1).Info("got a virtual machine event")
 
-	if req.AdmissionRequest.Operation == admissionv1beta1.Create {
-		err = a.mutateCreateVirtualMachinesFn(virtualMachine, logger)
-		if err != nil {
-			return admission.Errored(http.StatusInternalServerError,
-				fmt.Errorf("Failed to create virtual machine allocation error: %v", err))
-		}
-	} else if req.AdmissionRequest.Operation == admissionv1beta1.Update {
-		err = a.mutateUpdateVirtualMachinesFn(virtualMachine, logger)
-		if err != nil {
-			return admission.Errored(http.StatusInternalServerError,
-				fmt.Errorf("Failed to update virtual machine allocation error: %v", err))
+	if *req.DryRun != true {
```

Comment:
are we 100% sure that DryRun cannot be `nil` ?

Reply:
mmm. it's `omitempty` so I guess you're right. better safe than future bug